src/boot/efi/meson.build

```diff
@@ -258,6 +258,9 @@ if have_gnu_efi
                        '-shared',
                        '-Bsymbolic',
                        '-nostdlib',
+                       '--no-undefined',
+                       '--warn-common',
```

Comment:
hmm, we already set `-fno-common` for the whole build system, no?

Reply:
IIUC, using `--warn-common` with the linker is still reasonable, because it might catch anything that slips through.